runner/src/server/services/cacheService.ts

```diff
@@ -33,6 +33,7 @@ if (vcapServices) {
   const vcapJson = JSON.parse(vcapServices);
   if ("redis" in vcapJson) {
     redisUri = vcapJson.redis[0].credentials.uri;
+    console.log("Vcap Redis cache bound at: " + redisUri);
```

Comment:
Could we use the logger instead e.g. https://github.com/communitiesuk/digital-form-builder/blob/f5873d125a84ae8d6520bcaacf41200ae41e25ba/designer/server/lib/persistence/s3PersistenceService.ts#L29-L31